<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>290</width>
      <height>416</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="title_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Главное меню</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="airport_button">
       <property name="text">
        <string>Аэропорты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="passenger_button">
       <property name="text">
        <string>Пассажиры</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="staff_button">
       <property name="text">
        <string>Персонал</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="flight_button">
       <property name="text">
        <string>Авиарейсы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ticket_button">
       <property name="text">
        <string>Авиабилеты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="purpose_button">
       <property name="text">
        <string>Назначения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_button">
       <property name="text">
        <string>Загрузить данные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Сохранить изменения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Отменить последнее действие</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>309</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
